release/nightly_tests/nightly_tests.yaml

```diff
@@ -207,3 +207,31 @@
   smoke_test:
     cluster:
       compute_template: shuffle/shuffle_compute_smoke.yaml  # Does not exist yet
+
+# Stress tests with many tasks
+- name: stress_test_replacement_group
+  cluster:
+    app_config: stress_tests/stress_tests_app_config.yaml
```

Comment:
We don't need the stress test cluster. The pg stress test just is running in a single node.